src/prefect/cli/agent.py

```diff
@@ -90,7 +94,19 @@ def start(agent_option, token, name, verbose, label, no_pull, base_url):
         --base-url, -b  TEXT    A Docker daemon host URL for a LocalAgent
         --no-pull               Pull images for a LocalAgent
                                 Defaults to pulling if not provided
+
+    \b
+    Fargate Agent Options:
+        Any of the configuration options outlined in the docs can be provided here
+        https://docs.prefect.io/cloud/agent/fargate.html#configuration
     """
+
+    # Split context
+    kwargs = dict()
+    for item in ctx.args:
+        item = item.replace("--", "")
+        kwargs.update([item.split("=")])
```

Comment:
Did you test this? This appears to create a list within a list - not sure if that's intentional or not

Reply:
Yeah this properly creates the kwargs see test `test_agent_start_fargate_kwargs_received`

e.g.
```
    kwargs = dict()
    for item in ["a=1", "b=2"]: # context args
        item = item.replace("--", "")
        kwargs.update([item.split("=")])

    print(kwargs)

    {'a': '1', 'b': '2'}
```

Reply:
```
In [1]: a = [1, 2]

In [2]: b = [[1, 2]]

In [3]: c = dict()

In [4]: c.update(a)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-82ccf1900a77> in <module>
----> 1 c.update(a)

TypeError: cannot convert dictionary update sequence element #0 to a sequence

In [5]: c.update(b)

In [6]: c
Out[6]: {1: 2}
```

Reply:
whoa, TIL!